apps/src/templates/alert.jsx

```diff
@@ -16,19 +18,28 @@ var Alert = React.createClass({
         position: 'absolute',
         zIndex: 1000,
         top: 45,
-        left: 350,
+        left: this.props.leftMargin,
```

Comment:
So... we want the notification to fill the width of the editor, not including the toolbox, but we're parenting it to `#codeWorkspace` because we don't want to reach down into blockly/droplet, right?

I'd kind of prefer that sort of layout rule live outside of this component - like we made a container at the width we needed, and this component could just fill the container width.  I'm not sure that's necessarily better, but right now it feels a _little_ like magic knowledge of the thing we're being rendered above, which is weird.